commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -123,7 +124,7 @@ public void testGetStringFontIcon_returnsCorrectString() {
 
         String stringResource = TransferMethodUtils.getStringFontIcon(mContext,
                 HyperwalletTransferMethod.TransferMethodTypes.PAPER_CHECK);
-        assertThat(stringResource, is("paper icon"));
+        MatcherAssert.assertThat(stringResource, Matchers.is("paper icon"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed